### DISCUSSION TITLE
Excluding libraries from home screen

### DISCUSSION THREAD

**Discussion by ablackshear** [croneter/PlexKodiConnect#1524] [Q&A] [answered]
I have a few libraries for the kids and I'd like those to not show up in recent movies/episodes. In Plex I can set those to not show up on the home screen, but PKC doesn't use that. Is that something Plex exposes that could be used here?

**Comment by croneter** [comment, marked as answer]
PKC and Kodi offer MUCH more customization. You need to tweak your Kodi skin. See https://github.com/croneter/PlexKodiConnect/wiki/Video-Nodes-and-Skins

**Reply by ablackshear** [reply]
Exactly what I needed, didn't realize it was a skin thing, thanks
